mkdocs/utils/__init__.py

```diff
@@ -145,11 +153,12 @@ def clean_directory(directory):
             os.unlink(path)
 
 
-def copy_media_files(from_dir, to_dir, exclude=None, dirty=False):
+def copy_media_files(from_dir, to_dir, exclude=markdown_patterns, dirty=False):
```

Comment:
Here you can use a list comprehension to convert the list of extensions to patterns:

```python
 exclude=['*{0}'.format(x) for x in markdown_extensions]
```